docs/src/snap/howto/dualstack.md

```diff
@@ -0,0 +1,133 @@
+# How to enable Dual-Stack networking
+
+Dual-stack networking allows Kubernetes to support both IPv4 and IPv6 addresses
+simultaneously. This means that your pods, services can be assigned
+both IPv4 and IPv6 addresses, allowing them to communicate over either protocol.
+This document will guide you through enabling dual-stack, including necessary
+configurations, known limitations, and common issues.
+
+
+### Prerequisites
+
+Before enabling dual-stack, ensure that your environment supports IPv6, and
+that your network configuration (including any underlying infrastructure) is
+compatible with dual-stack operation.
+
+### Enabling Dual-Stack
+
+Dual-stack can be enabled by specifying both IPv4 and IPv6 CIDRs during the
+cluster bootstrap process. The key configuration parameters are:
+
+- **Pod CIDR**: Defines the IP range for pods.
+- **Service CIDR**: Defines the IP range for services.
+
+1. **Bootstrap Kubernetes with Dual-Stack CIDRs**
+
+   Start by creating a new container (e.g., using LXC) and ensure that IPv6
```

Comment:
I'm a little concerned about using LXC for this because of all the profile shenanigans and it presupposes that the LXD init enabled ipv6 doesn't it?

Reply:
I mean. I would leave it out entirely. You could include a link to setting up a node on LXD or Multipass or whatever if necessary.

Reply:
Do LXD/Multipass have a page explaining how to enable ipv6? Maybe we could simply link it? 
If not, it might be good to keep the section to be an "easy" follow along for a working example.

Reply:
@evilnick More of a general question, could our docs be more uniform using either lxd or multipass but not both for all our tutorials? Alternatively, we could have the alternative in a second tab for all tutorials. Are our users usually familiar with both LXD and multipass or just one?

Reply:
Let's leave it out entirely as Nick suggests. LXD should be setup automatically with IPv6 if the host supports it (same for multipass IIRC). Having that prerequisite should be enough.